schema/config-descriptor.json

```diff
@@ -0,0 +1,24 @@
+{
+  "description": "OpenContainer Content Descriptor Specification",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://opencontainers.org/schema/descriptor",
+  "type": "object",
+  "properties": {
+    "mediaType": {
+      "type": "string",
+      "pattern": "^application/vnd[.]oci[.]image[.]config[.]v1[+]json$"
```

Comment:
Ah, descriptors [can point at anything in general][1], but *this* is a descriptor that points at a config file.  That makes sense to me.

[1]: https://github.com/opencontainers/image-spec/pull/341#issuecomment-249217476